client/client.go

```diff
@@ -21,11 +20,7 @@ var _ Client = &client{}
 
 type Client interface {
 	// info.Client methods
-	IsBootstrapped(context.Context, string, ...rpc.Option) (bool, error)
-	GetNetworkName(context.Context, ...rpc.Option) (string, error)
-	Peers(context.Context, ...rpc.Option) ([]info.Peer, error)
-	GetNetworkID(context.Context, ...rpc.Option) (uint32, error)
-	GetBlockchainID(context.Context, string, ...rpc.Option) (ids.ID, error)
+	InfoClient
```

Comment:
just a minor consolidation, which I thought was necessary but turned out it really isn't. Still it should not be bad